app/controllers/users/registrations_controller.rb

```diff
@@ -16,6 +16,11 @@ def create
 
     if resource.save
       if resource.active_for_authentication?
+        last_ip = request.env['REMOTE_ADDR']
```

Comment:
I see this same logic block repeated in session_controller below, and I have something similar for logs in https://github.com/inaturalist/inaturalist/blob/master/config/initializers/logstasher.rb#L42 . Maybe we could extract this into a shared method. Not a blocker for merge - something for later